tests/StrictQuoteLineTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace ClickHouseDB\Tests;
+
+use PHPUnit\Framework\TestCase;
+use ClickHouseDB\Quote\StrictQuoteLine;
+
+/**
+ * Class StrictQuoteLineTest
```

Comment:
redundant comment, we can remove it as it doesn't add any value

we already know `class`name and `namespace`

Reply:
I remove comment, but all classes in test folder have comments

Reply:
Yup. I'm removing them but that's a lot of work. Currently I managed to remove most from /src folder.